lib/dom.dart

```diff
@@ -323,9 +318,9 @@ class Document extends Node
   int get nodeType => Node.DOCUMENT_NODE;
 
   // TODO(jmesserly): optmize this if needed
-  Element get documentElement => querySelector('html');
-  Element get head => documentElement.querySelector('head');
-  Element get body => documentElement.querySelector('body');
+  Element get documentElement => querySelector('html')!;
+  Element get head => documentElement.querySelector('head')!;
```

Comment:
At least `head` and `body` here should be nullable I am pretty sure, unless we are implementing some strict form of html?

Reply:
Hmm, yeah I agree with that. I think it's even safer for `documentElement` to be nullable as well. If this turns out to be inconvenient I think we could make it non-nullable in the next major version.